transmogrifier/sources/ead.py

```diff
@@ -117,6 +118,86 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
 
         # format field not used in EAD
 
+        # holdings
+        for holding_element in collection_description.find_all(
+            "originalsloc", recursive=False
+        ):
+            if holding_value := self.create_string_from_mixed_value(
+                holding_element, " ", ["head"]
+            ):
+                fields.setdefault("holdings", []).append(
+                    timdex.Holding(note=holding_value)
+                )
+        for holding_element in collection_description_did.find_all(
+            "physloc", recursive=False
+        ):
+            if holding_value := self.create_string_from_mixed_value(
+                holding_element, " ", ["head"]
+            ):
+                fields.setdefault("holdings", []).append(
+                    timdex.Holding(location=holding_value)
+                )
+
+        # identifiers
+        for id_element in collection_description_did.find_all("unitid"):
+            if id_value := self.create_string_from_mixed_value(
+                id_element,
+                " ",
+            ):
+                fields.setdefault("identifiers", []).append(
+                    timdex.Identifier(
+                        value=id_value,
+                    )
+                )
+
+        # languages
+        for language_element in collection_description_did.find_all(
+            "langmaterial", recursive=False
+        ):
+            for language_value in self.create_list_from_mixed_value(language_element):
+                if language_value not in ",.":
+                    fields.setdefault("languages", []).append(language_value)
+
+        # links, omitted pending decision on duplicating source_link
+
+        # literary_form field not used in EAD
+
+        # locations
+        for control_access_element in collection_description.find_all(
+            "controlaccess", recursive=False
+        ):
+            for location_element in control_access_element.find_all("geogname"):
+                if location_value := self.create_string_from_mixed_value(
+                    location_element,
+                    " ",
+                ):
+                    fields.setdefault("locations", []).append(
+                        timdex.Location(value=location_value)
+                    )
+
+        # notes
+        for note_element in collection_description.find_all(
```

Comment:
I think the notes field could use more consideration:

- For some if not all of these, instead of concatenating all of the paragraphs into strings I think it makes more sense to just append each of the `<p>` tag contents to the `note.value` list. 
- `bibliography` appears to have a different structure but definitely lends itself better to a list than a single string.
- It looks like the contents of the `<head>` tag might be a better fit for the `note.kind` than the crosswalked type value for some of these.
- I'm not sure the data in `acqinfo`, `appraisal`, `custodhist`, or `processinfo` seem useful for discovery purposes? Let's review.

Reply:
1. I'm creating a list of the `p` tags for `bioghist` and `scopecontent` but I still think we should concatenate the contents of each `p` tag given the likelihood of subelements. Let me know if you agree
2. Separating out `bibliography` from the other notes given the different structure and I'm updating the fixture for a better representation of our local data. 
3. I'm OK doing it this way and you can review how I did it but I think this adds complexity for largely insignificant changes in the `kind` value for the remaining notes `bibliography`, `bioghist`, and `scopecontent`. I scanned our data and the changes would be a few specific `bibliography` values with a person's name, `bioghist` might be `Biographical Note` or `Historical Note` rather than `Biography or History`, and the equivalent variation for `scopecontent`. It also opens up the risk of the `head` element being missing (however unlikely) and then we have a unspecified note as you can see in the updated attribute and subfield variations test. But I defer to you on whether that's a meaningful difference to users. Also, there's duplicative code here that I'll clean up after we confirm the approach
4. Looking at the data again (and as you can tell by #3), I agree!

Reply:
1. Yeah making the contents of each individual `<p>` tag a string makes sense.

2. Similar to the other notes with the `<p>` tag, it looks like for bibliography each element in the list should be the concatenated contents of a `<bibref>` tag. The bibliography notes are essentially a list of bibliographic references, so each of those should be an element in our `note.value` list. Right now the different parts of each bibliographic reference are getting split into separate list items, which doesn't really make sense. 

3. For the `kind` value, I don't think a few extra lines of code is significant added complexity and the `<head>` element value is more accurate, even if only slightly. We can easily fall back to the crosswalked type value if there's no `<head>` element to eliminate that problem, e.g.:
```
note_head_element = note_element.find("head", string=True)
note.kind = (
    note_head_element.string
    if note_head_element
    else self.crosswalk_type_value(note_element.name)
)
```

Reply:
1. 👍 
2. Done!
3. 👍 I also refactored this block to minimize duplicate code